cockroachcloud/create-a-serverless-cluster.md

```diff
@@ -55,13 +63,15 @@ Your cluster will be created in a few seconds.
 
 <section class="filter-content" markdown="1" data-scope="paid">
 
-1. Enter your **Spend limit**.
-
-    This is the maximum amount you could be charged per month. You will be charged only for what you use.
+1. If the option to **Start for free** is still available to you, select **Upgrade your resources** instead.
+    
+1. Enter your **Resource limits**.
+    - If you select **Set a monthly limit**, you can set storage and RU limits individually, or enter a dollar amount that will be split automatically between both resources. You will only be charged for the resources you use.   
+    - If you select **Unlimited**, your cluster will scale to meet your application's needs. You will only be charged for the resources you use.
 
-1. Click **Next: Payment info**.
+1. Click **Next: Payment**.
 
-1. Verify your cluster configuration and spend limit.
+1. Verify your cluster configuration resource limits.
```

Comment:
Suggest linking "resource limits" to the definition of that term from _includes/common/basic-terms.md (whichever page has that include and makes sense)